code/object_detector/algorithms/object_fill.py

```diff
@@ -44,22 +49,41 @@ def __init__(self,
         self.debug = debug
         self._last_center: Optional[Vector] = None
         self._dynamic_fill_size = dynamic_fill_size
+        self._blacklisted_pixels = {None}
 
     def locate_center(self, frame: np.ndarray) -> Optional[Vector]:
         """
         :param frame: The image to search in
         :return: The center of the object (Can be null).
         """
-        image_size = Vector(int(frame.shape[1]), frame.shape[0])
 
+        image_size = Vector(frame.shape[1], frame.shape[0])
+
+        # """
+        new_center = None
+        for i in range(10):
```

Comment:
Magic number, also hvorfor capper vi egentlig overhovedet på 10? Føler lidt vi på det her tdispunkt anyway bare burde søge hele billedet igennem, i stedet for at cappe på 10 objekter? Eller?

Reply:
(Hvis det er tilfældet, at vi skal søge hele billedet igennem kan locate object funktionen optimeres en del)

Reply:
hvordan tænker du at optimere det? :) skal vi smide *alt* i blacklist?